src/StandbyPool/StandbyPool.Autorest/README.md

```diff
@@ -122,28 +144,73 @@ directive:
     set:
       parameter-name: MaxReadyCapacity
 
+  - where:
+      verb: New|Update
+      subject: StandbyVMPool
+      parameter-name: ElasticityProfileMinReadyCapacity
+    set:
+      parameter-name: MinReadyCapacity
+
   - where:
       verb: New|Update
       subject: StandbyVMPool
       parameter-name: VirtualMachineState
     set:
       parameter-name: VMState
 
-# Rename standby virtual machine parameters
   - where:
       verb: Get
       subject: StandbyVMPoolVM
       parameter-name: Name
     set:
       parameter-name: VMName
-      
+
   - where:
       verb: Get
       subject: StandbyVMPoolVM
       parameter-name: PoolName
     set:
       parameter-name: Name
 
+  - where:
+      verb: Get
+      subject: StandbyVMPoolStatus
+      parameter-name: StandbyVirtualMachinePoolName
+    set:
+      parameter-name: Name
+
+  - where:
+      verb: Get
+      subject: StandbyVMPoolStatus
+      parameter-name: RuntimeView
+    set:
+      parameter-name: Version
+
+  # Hide parameters from cmdlets
+  - where:
+      verb: Get
+      subject: StandbyVMPoolStatus
+      parameter-name: Version
+    hide: true
+    set:
+      default:
+        script: "'latest'"
+
+  - where:
+      verb: Get
+      subject: StandbyContainerGroupPoolStatus
+      parameter-name: Version
+    hide: true
+    set:
+      default:
+        script: "'latest'"
+
+  # Rename Get-StandbyVirtualMachine to StandbyVM
```

Comment:
Based on the comments, it should be an operation of renaming. But actually, it is to remove the cmdlet. Is it expected? Besides this cmdlet is exported in the last version.

Reply:
Hi @dolauli, it's expected, as this cmdlets is not functioning for right now.

Reply:
> Hi @dolauli, it's expected, as this cmdlets is not functioning for right now.

@fangru2023 The issue is the comment should be consistant with the actual directives

Reply:
Addressed comments. Thanks for suggestion.